main.go

```diff
@@ -136,19 +138,6 @@ func mainWithError() error {
 
 	daemonCommand := newCommand.DaemonCommand().CobraCommand()
 
-	daemonCommand.PersistentFlags().String(f.Service.Kubernetes.Address, "http://127.0.0.1:6443", "Address used to connect to Kubernetes. When empty in-cluster config is created.")
-	daemonCommand.PersistentFlags().Bool(f.Service.Kubernetes.InCluster, false, "Whether to use the in-cluster config to authenticate with Kubernetes.")
-	daemonCommand.PersistentFlags().String(f.Service.Kubernetes.TLS.CAFile, "", "Certificate authority file path to use to authenticate with Kubernetes.")
-	daemonCommand.PersistentFlags().String(f.Service.Kubernetes.TLS.CrtFile, "", "Certificate file path to use to authenticate with Kubernetes.")
-	daemonCommand.PersistentFlags().String(f.Service.Kubernetes.TLS.KeyFile, "", "Key file path to use to authenticate with Kubernetes.")
-
-	daemonCommand.PersistentFlags().String(f.Service.Installer.Helm.HelmBinaryPath, "/bin/helm", "Path to Helm binary. Needs CNR registry plugin installed.")
-	daemonCommand.PersistentFlags().String(f.Service.Installer.Helm.Organisation, "", "Organisation of Helm CNR registry.")
-	daemonCommand.PersistentFlags().String(f.Service.Installer.Helm.Password, "", "Password for Helm CNR registry.")
-	daemonCommand.PersistentFlags().String(f.Service.Installer.Helm.Registry, "quay.io", "URL for Helm CNR registry.")
-	daemonCommand.PersistentFlags().String(f.Service.Installer.Helm.Username, "", "Username for Helm CNR registry.")
-	daemonCommand.PersistentFlags().String(f.Service.Installer.Type, string(helm.HelmInstallerType), "Which installer to use for installation management.")
-
```

Comment:
Boyscouting the order here.

Reply:
It's depressing how bad i was at the ordering of these :D